.github/workflows/ios_deploy_staging_to_firebase.yml

```
@@ -20,6 +20,7 @@ jobs:
       FIREBASE_CLI_TOKEN: ${{ secrets.FIREBASE_CLI_TOKEN }}
       FIREBASE_APP_ID_STAGING: ${{ secrets.FIREBASE_IOS_APP_ID_STAGING }}
       FIREBASE_DISTRIBUTION_TESTER_GROUPS: ${{ vars.FIREBASE_DISTRIBUTION_TESTER_GROUPS }}
+      GITHUB_RUN_NUMBER: ${{ secrets.GITHUB_RUN_NUMBER }}
```

Wait, `GITHUB_RUN_NUMBER` is not a secret 🤔 https://github.com/nimblehq/flutter-templates/blob/feature/241-ios-github-run-number/.github/workflows/android_deploy_staging.yml#L53

```suggestion
      GITHUB_RUN_NUMBER: $GITHUB_RUN_NUMBER
```

We can edit the `branches:` trigger to have this branch `feature/241-ios-github-run-number` for verifying. After verifying the CD works properly, we can revert the trigger on this CD workflow 🙏

---

as `GITHUB_RUN_NUMBER` is a normal env variable. We even don't need to pass it to Fastlane as Fastlane runs on the GitHub Linux runner env. 🤔 

```suggestion
```

Again, please test it 🙏